test/account_api_test.rb

```diff
@@ -0,0 +1,278 @@
+require "test_helper"
+require "gds_api/account_api"
+
+describe GdsApi::AccountApi do
+  include PactTest
+
+  let(:api_client) { GdsApi::AccountApi.new(account_api_host) }
+
+  let(:authenticated_headers) { { "GOVUK-Account-Session" => govuk_account_session } }
```

Comment:
Given the travails in testing these over in frontend, might this be better as a standard capitalised + HTTP_ prefixed name?

Reply:
```suggestion
  let(:authenticated_headers) { { "HTTP_GOVUK_ACCOUNT_SESSION" => govuk_account_session } }
```

Reply:
No, the request header is `GOVUK-Account-Session` - the header-name-mangling is done by Rails: see https://github.com/alphagov/frontend/pull/2668#issuecomment-802843544

Reply:
ah ha, great - cheers for the link!